kolibri/plugins/video_mp4_render/assets/src/vue/index.vue

```diff
@@ -74,14 +78,17 @@
       },
 
       setPlayState(state) {
+        this.recordProgress();
         if (state === true) {
           this.videoPlayer.$('.videotoggle').classList.add('videopaused');
           this.videoPlayer.$('.videoreplay').classList.add('display');
           this.videoPlayer.$('.videoforward').classList.add('display');
+          this.$emit('startTracking', this.Kolibri);
         } else {
           this.videoPlayer.$('.videotoggle').classList.remove('videopaused');
           this.videoPlayer.$('.videoreplay').classList.remove('display');
           this.videoPlayer.$('.videoforward').classList.remove('display');
+          this.$emit('stopTracking', this.Kolibri);
```

Comment:
See my comment on how to resolve this: https://github.com/learningequality/kolibri/pull/374/files#r74622627